src/error-handlers/500.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = function(err, response, request, next){
```

Comment:
-1 point: There is a slight error in here, you have response before request that needs to be changed because it always needs to be `request` and then `response`